plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnector.java

```diff
@@ -1318,9 +1319,15 @@ private ContainerInfo createContainerInfo(Service svc,
         info.setNetworkSettings(networkSettings);
         info.setHostConfig(hostConfig);
         info.setImage(imageInfo.getConfig().getImage());
+        info.setState(getContainerState(pod));
         return info;
     }
 
+    private ContainerState getContainerState(final Pod pod) {
+        ContainerState state = new ContainerState();
+        state.setStatus(pod.getStatus().getPhase());
```

Comment:
@sunix pod's states are "Pending / Running / Succeeded / Failed / Unknown" 
What about matching it the following way:
1. created = Succeeded
2. running = Running 
3. dead / exited = Failed